New Term: Aggressive Scan (#1280)
Codecademy/docs: content/cybersecurity/concepts/nmap/terms/aggressive-scan/aggressive-scan.md

---
Title: 'Aggressive Scan'

> Description: 'An aggressive scan provides far better information than a standard scan, but is more likely to be detected.'

Reviewer: ```suggestion
Description: 'Provides far better information than a standard scan, but is more likely to be detected.'
```

Subjects:
  - 'Computer Science'
  - 'Networking'
Tags:
  - 'Networking'
  - 'Cybersecurity'
  - 'Ethical Hacking'
CatalogContent:
  - 'introduction-to-cybersecurity'
  - 'paths/scan-systems-with-nmap'
---

> An **aggressive scan** provides far better information than a [regular scan](https://www.codecademy.com/resources/docs/cybersecurity/nmap/nmap-scan), but is more likely to be detected. It is performed by using the `-A` option. The aggressive scan enables OS detection, version detection, script scanning, and traceroute. It sends out more probes than a regular scan, so it is more likely to be detected during a security audit.

Reviewer: ```suggestion
An **aggressive scan** provides far better information than a [regular scan](https://www.codecademy.com/resources/docs/cybersecurity/nmap/nmap-scan), but is more likely to be detected. It is performed by using the `-A` option and enables the following:

- OS detection (`-o`)
- Version detection (`-sV`)
- Script scanning (`-sC`)
- Traceroute (`--traceroute`)

Aggressive scans send out more probes than a regular scan, and are more likely to be detected during a security audit.
```

## Syntax

```pseudo
nmap -A <target>
```

The above conducts an aggressive scan on `<target>`.

## Example

> The following example runs an aggressive scan on the site `scanme.nmap.org`:

Reviewer: ```suggestion
The following example runs an aggressive scan on the site [scanme.nmap.org](http://scanme.nmap.org/):
```

> ````shell

Reviewer: ```suggestion
```shell
```

nmap -A scanme.nmap.org

> 

Reviewer: We're missing closing backticks here.

> > **Note:** The site `scanme.nmap.org` is a site explicitly set up for testing Nmap. Only perform scans on domains you have explicit permission to scan.

Reviewer: ```suggestion
> **Note:** This is a site explicitly set up for testing Nmap. Scans should only be performed on domains with explicit permissions given to the user.
```
